ui/src/dashboards/utils/cellGetters.js

```diff
@@ -89,7 +89,7 @@ export const getNewDashboardCell = (dashboard, cellType) => {
 export const getClonedDashboardCell = (dashboard, cloneCell) => {
   const {x, y} = getNextAvailablePosition(dashboard, cloneCell)
 
-  const name = `${cloneCell.name} (Clone)`
+  const name = `${cloneCell.name} (clone)`
```

Comment:
was it intentional to change `(clone)` from upper to lowercase?

Reply:
yes, it matches the dashboards, and I think looks better